ibr/ibr-sdk/build/ibr-sdk.js

```diff
@@ -56,7 +66,11 @@
         }
 
         // Render data into three.js objects
-        var materials = [new THREE.LineBasicMaterial( { color: 0x00ffff } ), new THREE.LineBasicMaterial( { color: 0xff0000 } )];
+        var materials1 = new THREE.LineBasicMaterial( { color: 0xff0000 } );
```

Comment:
nit: more descriptive naming

also, materials looks like just a list of duplicated linebasicsmaterial, this might be what's slowing down the renderer at scale.

can we create these on demand in L83 instead? or even better, can we create one of each and then reuse?

Reply:
Removed manually defined material objects and use randomly generate colors from pre-defined distinguishable selection of colors.